src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const reversedRobot = {};
+
+  for (const key in robot) {
+    if (Object.keys(reversedRobot).includes(robot[key])) {
```

Comment:
`Object.keys()` use loop inside for creating an array of keys. Let's try to check the existence of the key without using extra loops